.github/workflows/release.yml

```
@@ -64,46 +43,15 @@ jobs:
-          chmod 600 android/upload-keystore.jks
-          echo "Keystore decoded and permissions set"
+          flutter doctor -v
 
-      - name: Create key.properties
-        env:
-          KEYSTORE_PASSWORD: ${{ secrets.KEYSTORE_PASSWORD }}
-          KEY_PASSWORD: ${{ secrets.KEY_PASSWORD }}
-          KEY_ALIAS: ${{ secrets.KEY_ALIAS }}
+      - name: Check submodule
+        id: check_submodule
         run: |
```

actionlint (1.7.12)

[error] property "checkout" is not defined in object type {beta: bool}

(expression)